googletest/PSPBUILD

```diff
@@ -0,0 +1,43 @@
+pkgname=googletest
+pkgver=1.14.0
+pkgrel=1
+pkgdesc="GoogleTest - Google Testing and Mocking Framework"
+arch=('mips')
+url="google.github.io/googletest/"
+license=('BSD')
```

Comment:
I think we could be a bit more specific here and set it to BSD-3-Clause

Reply:
Done!